components/xsd-fu/templates-cpp/OMEXMLModelObject.template

```diff
@@ -994,7 +1071,7 @@ ${customUpdatePropertyContent[prop.name]}
       }
 {% end source %}\
 {% end %}\
-{% when prop.maxOccurs > 1 and not klass.isAbstract and not klass.isAbstractProprietary %}
+{% when prop.maxOccurs > 1 and not klass.isAbstract %}\
```

Comment:
I'll look over this in more detail tomorrow.  As a general rule, every conditional uses `%}\` to end every line to avoid any trailing characters contributing to the output, including newlines, so that every line in the output is explicit and obvious.  This was due to some subtle problems in the past.